src/patient-registration/patient-registration-helper.tsx

```diff
@@ -28,11 +29,13 @@ export interface PatientIdentifier {
 }
 
 export type Patient = {
+  uuid: string;
   identifiers: Array<PatientIdentifier>;
   person: {
+    uuid: string;
     names: Array<NameValue>;
     gender: string;
-    birthdate: Date;
+    birthdate: string;
```

Comment:
> In this case is it necessary? I don’t think so.
> On Wed, 16 Sep 2020 at 10:24 Samuel Male ***@***.***> wrote: ***@***.**** commented on this pull request. ------------------------------ In src/patient-registration/patient-registration.component.tsx <[#46 (comment)](https://github.com/openmrs/openmrs-esm-patient-registration/pull/46#discussion_r489294054)> : > @@ -256,18 +357,21 @@ export const PatientRegistration: React.FC = () => { {props => ( <Form className={styles.form}> <div className={styles.formTitle}> - <h1 className={`omrs-type-title-1 ${styles.title}`}>New Patient</h1> - {localStorage.getItem('openmrs:devtools') === 'true' && <DummyDataInput setValues={props.setValues} />} + <h1 className={`omrs-type-title-1 ${styles.title}`}>{!!existingPatient ? 'Edit' : 'New'} Patient</h1> It's the same this actually. The valid cases to use such a pattern is when you want to assign a boolean value for a property that expects a boolean with Typescript. It's literately coming up with an explicit expression that returns a boolean. — You are receiving this because you commented. Reply to this email directly, view it on GitHub <[#46 (comment)](https://github.com/openmrs/openmrs-esm-patient-registration/pull/46#discussion_r489294054)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACWOHPREZF363PFI34D6O3LSGB73BANCNFSM4RNXW6MA> .
> -- - Larry It's not necessary.

It's not necessary

Reply:
I don't understand the response lol.

Reply:
>I don't understand the response lol.

Sorry, it was a mistake, I deleted it.